src/core/library.c

```diff
@@ -1104,6 +1178,27 @@ QuicLibraryGetGlobalParam(
         Status = QUIC_STATUS_SUCCESS;
         break;
 
+    case QUIC_PARAM_GLOBAL_RAW_DATAPATH_PROCS:
+
+        if (*BufferLength == 0 ||
+            *BufferLength < sizeof(uint16_t) * MsQuicLib.RawDataPathProcListLength) {
+            *BufferLength = sizeof(uint16_t) * MsQuicLib.RawDataPathProcListLength;
+            Status = QUIC_STATUS_BUFFER_TOO_SMALL;
```

Comment:
What is the list is zero? Wouldn't an input of 0 be STATUS_SUCCESS instead of TOO_SMALL?

Reply:
Now returns sucess if the list is zero.